Login/js/new.js

```diff
@@ -85,23 +85,35 @@ var callbacks = {
    * Submit the posting form and await the return of the post.
    */
   submitCredentials: function() {
-    $("#save").prop('disabled', true);
-    privlyNetworkService.sameOriginPostRequest(
-      privlyNetworkService.contentServerDomain() + "/users/sign_in", 
-      callbacks.checkCredentials,
-      {"user[email]":  $("#user_email").val(),
-       "user[password]":  $("#user_password").val()
-       });
+    
+    var userEmail = $("#user_email").val();
+    var userPassword = $("#user_password").val();
+    if (userEmail != "" && userPassword != "") {
+      $("#save").prop('disabled', true);
```

Comment:
You should refactor this into an event that fires on keydown and submit and checks for contents in the form. I think it is possible to permanently lock the form if they submit the wrong password/email combo, no?